JPF/import-dob-permits.py

```diff
@@ -0,0 +1,179 @@
+        'HIC License':                            'object',
+        'Site Safety Mgr\'s First Name':            'object',
+        'Site Safety Mgr\'s Last Name':             'object',
+        'Site Safety Mgr Business Name':          'object',
+        'Superintendent First And Last Name':     'object',
+        'Superintendent Business Name':           'object',
+        'Owner\'s Business Type':                   'object',
+        'Non-profit':                             'object',
+        'Owner\'s Business Name':                   'object',
+        'Owner\'s First Name':                      'object',
+        'Owner\'s Last Name':                       'object',
+        'Owner\'s House #':                       'object',
+        'Owner\'s House Street Name':               'object',
+        'Owner\'s House City':                      'object',
+        'Owner\'s House State':                     'object',
+        'Owner\'s House Zip Code':                  'object',
+        'Owner\'s Phone #':                       'object',
+        'DOBRunDate':                             'object'
+        }
+
+
+perm_dob_df_keep_cols = [
+        'BOROUGH',
+        'Bin #',
+        'House #',
+        'Street Name',
+        'Job #',
+        'Job doc. #',
+        'Job Type',
+        'Block',
+        'Lot',
+        'Zip Code',
+        'Bldg Type',
+        'Residential',
+        'Work Type',
+        'Permit Status',
+        'Filing Status',
+        'Permit Type',
+        'Filing Date',
+        'Issuance Date',
+        'Expiration Date',
+        'Job Start Date',
+        'DOBRunDate'
+        ]
+
+
+perm_dob_date_time_columns = [
+        'filing_date',
+        'issuance_date',
+        'expiration_date',
+        'job_start_date',
+        'dobrundate'
+        ]
+
+
+perm_dob_truncate_columns = ['borough']
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description='Import hpd buildings dataset.')
+    parser = add_common_arguments(parser)
+    args = parser.parse_args()
+
+    print args
+
+    if not args.SKIP_IMPORT:
+        import_csv(args)
+
+    sql_cleanup(args)
+
+
+def import_csv(args):
+    dob_permits_dir = os.path.join(BASE_DIR, DOB_PERMITS_KEY)
+    mkdir_p(dob_permits_dir)
+
+    dob_permits_csv = os.path.join(dob_permits_dir, "dob_permits.csv")
+
+    if not os.path.isfile(dob_permits_csv) or args.BUST_DISK_CACHE:
+        log.info("DL-ing DOB Permits")
+        download_file(
+                'https://data.cityofnewyork.us/api/views/ipu4-2q9a/rows.csv?accessType=DOWNLOAD',
+                dob_permits_csv)
+    else:
+        log.info("DOB Permits exists, moving on...")
+
+    perm_dob_description = 'DOB Permits'
+    perm_dob_pickle = dob_permits_dir + '/df_dob_permit.pkl'
+    perm_dob_sep_char = ","
+    perm_dob_table_name = "dob_permits"
+    perm_dob_load_pickle = args.LOAD_PICKLE
+    perm_dob_save_pickle = args.SAVE_PICKLE
+    perm_dob_db_action = "replace"
```

Comment:
We should use the `--update-or-replace-db` [flag](https://github.com/chrishenry/deltanyc-heatseek/blob/master/JPF/utils.py#L38)